### DISCUSSION TITLE
Preventing Windows from prompting about security risks about downloading MAPEO

### DISCUSSION THREAD

**Discussion by rudokemper** [digidem/mapeo-desktop#599] [Ideas]
**Describe the idea**
Currently, when downloading MAPEO on Windows, the browser prompts the user with a notification that MAPEO may harm the system, and the way to continue downloading MAPEO anyway is a little obscure -- one has to click the link "Conservar de todos modos" (Save Anyway) in the screenshot below.

![Windows install](https://user-images.githubusercontent.com/31662219/101651244-d9050180-3a0a-11eb-8af0-fb758355d782.PNG)

Could there be a way to update our certificate that we use to sign Mapeo, to guarantee trust by Windows?  

**What's your role? Where and how do you currently use the application?**
I'm on the programs team with Digital Democracy, and I recently encountered this issue with one of our partners, who encountered the issue. (The above screenshot is from their laptop directly.)

**Comment by gmaclennan** [comment]
This is an issue with Microsoft code signing certificates. According to [this Stackoverflow post](https://stackoverflow.com/a/51113771/3071863), when you have a "standard" code signing certificate, then this warning will show until "enough" users have installed and used you app, so that you gain a reputation. This reputation can be lost if the certificate is renewed. A workaround is to use an "EV" (Extended Validation) certificate, which is trusted immediately.

We currently sign with a standard certificate. It costs $75 per year. An EV certificate costs $280 per year. Our current certificate is due for renewal later this year.

Couple of options:

- Hope that as more users use Mapeo, then this warning will go away (but might persist on computers that are never online, to update their trusted list)
- Purchase the EV certificate.

**Comment by ximenabb** [comment]
Note to look into in Jan/Feb 2022
